### PR TITLE
Lexorin + Hypopen 1984

### PR DESCRIPTION
**Pull request by GoodWheatley**
## About the PR

Lexorin now requires Heartbreaker Toxin instead of ammonia to make, and you need to heat up the chemical mixture to make Mindbreaker Toxin. Lexorin is stupidly easy to make considering it's one of the best poisons in the game, so this fixes that. And in my neverending quest to get people to buy something other than the hypopen, I lowered the chem capacity of it to 10u so you have to stop and refill it to inject a lethal dose of Lexorin.

**Media**

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- tweak: After the Hypopen Incident of 2556, The recipe for Lexorin now requires Heartbreaker Toxin instead of Ammonia.
- tweak: You need to heat up the chemicals for Mindbreaker Toxin to get them to react.
- tweak: Due to budget cuts, the Syndicate is now offering an inferior version of the hypopen that only holds 10 units of chemicals.

### CODE REVIEW

**Comment by arealbeba** [comment]
there was no need to lower hypopen capacity tbh

**Comment by deltanedas** [comment]
i fully agree with making lex recipe meth-tier, but hypo nerf isnt needed
10u of lex can still crit and refilling can be done quickly if you really want a quicker kill

**Comment by GoodWheatley** [comment]
> 10u of lex can still crit and refilling can be done quickly if you really want a quicker kill
The reason for the tandem hypopen nerf is so that other chems used in the hypopen won't immediately kill the victim if it's all injected. By forcing the traitor to grab the bottle and refill the pen, the overall kill rate of the pen is slowed down and the victim has time to react, making it less effective at uncounterable instakills.

File: Resources/Prototypes/Recipes/Reactions/medicine.yml [hunk 1]
```diff
@@ -277,6 +277,7 @@
 
 - type: reaction
   id: MindbreakerToxin
+  minTemp: 370
```
Comment (deltanedas) [inline Resources/Prototypes/Recipes/Reactions/medicine.yml:280]: 37C is 310K, not 370K. did you not intend to use the temp from the removed todo?

**Comment by deltanedas** [comment]
also the argument of time to react doesn't matter, 10u of lexorin and 15u take the same time to crit, you can't do anything about it at that point so an extra jab won't make a difference

**Comment by GoodWheatley** [comment]
> also the argument of time to react doesn't matter, 10u of lexorin and 15u take the same time to crit, you can't do anything about it at that point so an extra jab won't make a difference

The point of the capacity nerf is that if they're using something that isn't lexorin, the hypopen user can't administer a lethal dose in one full use of the pen. They'll have to stop and refill the pen from the bottle, which makes "hey I think that guy has a hypopen" much more obvious to bystanders.

File: Resources/Prototypes/Recipes/Reactions/medicine.yml (hunk 1 above)
Comment (MagnusCrowe) [reply Resources/Prototypes/Recipes/Reactions/medicine.yml:280]: How does dexalin prevent lexorin poisoning?  It does not remove lexorin from the bloodstream like dexalin plus does.  It also only cures 2 asphix per unit while lexorin deals 14 per unit so it won't even have much of an effect at delaying it.

Unless I'm mistaken, won't this change make a lexorin/mindbreaker mix unstoppable?

**Pull request merged**